Staff Procedure for incident handling.md

```diff
@@ -1,65 +1,122 @@
 
- - Do not overtly invite them to withdraw the complaint or mention that withdrawal is OK. This suggests that you want them to do so, and is therefore coercive. "If you're OK with it [pursuing the complaint]" suggests that you are by default pursuing it and is not coercive.
- - Do not ask for their advice on how to deal with the complaint. This is a staff responsibility.
- - Do not offer them input into penalties. This is the staff's responsibility.
+What follows are examples of possible responses to an incident report. This list is is not inclusive, and PyCon reserves the right to take any action it deems necessary. Possible responses to an incident include:
 
-Once something is reported to a staff member, immediately meet with the conference chair and/or event coordinator. The main objectives of this meeting is to find out the following:
+* Nothing, if the behavior was determined to not be a Code of Conduct violation
+* A verbal or emailed warning
+* Requiring that the reported person avoid any interaction with, and physical proximity to, another person for the remainder of the event
+* Requiring the reported person not attend evening events
+* Refusal of alcoholic beverage purchases at events
+* Ending a talk that violates the Code of Conduct early
+* Not publishing the video or slides of a talk that violated the Code of Conduct
+* Not allowing a speaker who violated the Code of Conduct to give (further) talks at the event now or in the future
+* Immediately ending any event volunteer responsibilities and privileges the a person holds
+* Requiring that a person not volunteer for future events your organization runs (either indefinitely or for a certain time period)
+* Requiring that a person refund any travel grants and similar they received (this would need to be a condition of the grant at the time of being awarded)
+* Requiring that a person  immediately leave the event and not return
+* Banning a person from future events (either indefinitely or for a certain time period)
+* Removing a person  from membership of relevant organizations
+* Publishing an account of the incident and calling for the resignation of a person from their responsibilities (usually pursued by people without formal authority: may be called for if the person is the event leader, or refuses to stand aside from the conflict of interest, or similar, typically event staff have sufficient governing rights over their space that this isn't as useful)
 
- - What happened?
- - Are we doing anything about it?
- - Who is doing those things?
- - When are they doing them?
+Give reported person a place to appeal to if there is one. Keep in mind that it is not a good idea to encourage an apology from the harasser.
```

Comment:
"Give reported person a place to appeal to if there is one"
should be
"If a reported person wants to appeal the decision, they may contact the Python Software Foundation Board <psf@python.org>."

"Keep in mind that it is not a good idea to encourage an apology from the **harasser**."
should be
"Keep in mind that it is not a good idea to encourage an apology from the **reported person**."